ingest/src/main/java/com/connexta/ingest/adaptors/S3StorageAdaptor.java

```diff
@@ -42,16 +45,25 @@ public void store(StoreRequest storeRequest, String key) throws IOException {
     final String fileName = storeRequest.getFileName();
 
     LOGGER.info("Storing {} in bucket \"{}\" with key \"{}\"", fileName, s3BucketQuarantine, key);
-    s3Client.putObject(
-        PutObjectRequest.builder()
-            .bucket(s3BucketQuarantine)
-            .key(key)
-            .contentType(storeRequest.getMimeType())
-            .contentLength(storeRequest.getFileSize())
-            .metadata(ImmutableMap.of("filename", fileName))
-            .build(),
-        RequestBody.fromInputStream(
-            storeRequest.getFile().getInputStream(), storeRequest.getFile().getSize()));
+    try {
+      s3Client.putObject(
+          PutObjectRequest.builder()
+              .bucket(s3BucketQuarantine)
+              .key(key)
+              .contentType(storeRequest.getMimeType())
+              .contentLength(storeRequest.getFileSize())
+              .metadata(ImmutableMap.of("filename", fileName))
+              .build(),
+          RequestBody.fromInputStream(
+              storeRequest.getFile().getInputStream(), storeRequest.getFile().getSize()));
+
+    } catch (SdkClientException e) {
+      throw new StorageException(
```

Comment:
❓ Should we do the same thing on client fail and server fail?

Reply:
By client and server, what are you referring to?

Reply:
client exception vs. server exception. It's possible we don't have to do anything different, just wanted to make sure

Reply:
Unless I'm mistaken, the two exceptions we catch here should take care of it.